index.js

```diff
@@ -82,6 +86,15 @@ class ServerlessCustomDomain {
     this.targetHostedZoneName = this.givenDomainName.substring(this.givenDomainName.indexOf('.') + 1);
   }
 
+  setEndpointType(endpointType = 'EDGE') {
+    if (!SUPPORTED_ENDPOINT_TYPES.includes(endpointType)) throw new Error(`${endpointType} is not supported endpointType, use EDGE or REGIONAL.`);
+    this.endpointType = endpointType || 'EDGE';
```

Comment:
Can we drop the `|| 'EDGE'` since there's a default parameter?

Reply:
we definitely should, I was sure I removed it...